static/src/javascripts/projects/common/modules/commercial/adblock-banner-config.js

```diff
@@ -0,0 +1,127 @@
+                    quoteAuthor: 'Polly Toynbee',
+                    customCssClass: 'toynbee',
+                    imageAuthor: '//i.guim.co.uk/img/static/sys-images/Guardian/Pix/contributor/2014/6/30/1404146756739/Polly-Toynbee-L.png?w=300&amp;q=85&amp;auto=format&amp;sharp=10&amp;s=abf0ce1a1a7935e82612b330322f5fa4'
+                },
+                {
+                    supporterLink: 'https://membership.theguardian.com/supporter?INTCMP=ADBLOCK_BANNER_MACASKILL',
+                    quoteText: 'The Guardian enjoys rare freedom and independence. Support our journalism',
+                    quoteAuthor: 'Ewen MacAskill',
+                    customCssClass: 'macaskill',
+                    imageAuthor: '//i.guim.co.uk/img/static/sys-images/Guardian/Pix/contributor/2015/8/18/1439913873894/Ewen-MacAskill-R.png?w=300&q=85&auto=format&sharp=10&s=0ecfbc78dc606a01c0a9b04bd5ac7a82'
+                }
+            ],
+
+            editions: {
+                UK: {
+                    messageText: 'Become a supporter from just £5 per month to ensure quality journalism is available to all',
+                    trackingSuffix: '_UK'
+                },
+                US: {
+                    messageText: 'Become a supporter from just $4.99 per month to ensure quality journalism is available to all',
+                    trackingSuffix: '_US'
+
+                },
+                AU: {
+                    messageText: 'Become a supporter from just £5 per month to ensure quality journalism is available to all',
+                    trackingSuffix: '_AU'
+                },
+                INT: {
+                    messageText: 'Become a supporter from just $4.99 per month to ensure quality journalism is available to all',
+                    trackingSuffix: '_INT'
+                }
+            }
+        },
+        {
+            template: 'adblock-sticky-message-coin',
+
+            defaults: {
+                cursor: cursor,
+                customCssClass: '',
+                rectangleLogo: svgs('logomembership'),
+                marque36icon: svgs('marque36icon')
+            },
+
+            variants: [
+                {
+                    quoteText: 'Quality journalism',
+                    supporterLink: 'https://membership.theguardian.com/supporter?INTCMP=ADBLOCK_BANNER_COIN'
+                }
+            ],
+
+            editions: {
+                UK: {
+                    monthlyCost: '£5',
+                    dailyCost: '16p',
+                    adblockCoins: svgs('adblockCoinsUk')
+                },
+                INT: {
+                    monthlyCost: '$4.99',
+                    dailyCost: '16¢',
+                    adblockCoins: svgs('adblockCoinsUs')
+                }
+            }
+        }
+    ];
+
+
+    return {
+        banners: banners,
+        getBanners: function (edition) {
+            if (!edition) {
+                return [];
+            }
+            var out = [];
```

Comment:
You can use `return array.map(x => fx)` rather than `accumulator = []; array.forEach(x => {accumulator.push(x)}); return accumulator`.